k8s/nginx/development/global-config.yaml

```diff
@@ -56,6 +56,7 @@ data:
       set $ip_address $remote_addr;
       
       # Pass domain name and IP address as headers
-      proxy_set_header X-Domain-Name $domain_name;
-      proxy_set_header X-IP-Address $ip_address;
+      proxy_set_header Host $host;
```

Comment:
Hi @Psalmz777  , what is the difference here between: `proxy_set_header X-Forwarded-For $remote_addr` and `proxy_set_header X-Real-IP $remote_addr`?

I sense that we are assigning two different headers using the same value? Could I be missing something?

Reply:
Lemme slack you on this @Baalmart